### PR TITLE
Collection of changes requested during review of acceptance testing

### PR DESCRIPTION
**Pull request by purbon**
List of changes:

* Remove unused dependencies
* cleanup readme
* add new platforms to the list of options

### CODE REVIEW

File: qa/config/platforms.json [hunk 1]
```diff
@@ -3,14 +3,17 @@
...
     "centos-6": { "box": "elastic/centos-6-x86_64", "type": "redhat" },
     "centos-7": { "box": "elastic/centos-7-x86_64", "type": "redhat" },
     "oel-6": { "box": "elastic/oraclelinux-6-x86_64", "type": "redhat" },
     "oel-7": { "box": "elastic/oraclelinux-7-x86_64", "type": "redhat" },
     "fedora-22": { "box": "elastic/fedora-22-x86_64", "type": "redhat" },
+    "fedora-23": { "box": "elastic/fedora-23-x86_64", "type": "redhat" },
+    "debian-7": { "box": "elastic/debian-7-x86_64", "type": "debian", "specific":  true },
     "debian-8": { "box": "elastic/debian-8-x86_64", "type": "debian", "specific":  true },
     "opensuse-13": { "box": "elastic/opensuse-13-x86_64", "type": "suse" },
+    "sles-11": { "box": "elastic/sles-11-x86_64", "type": "suse", "specific": true }
```
Comment (untergeek) [inline qa/config/platforms.json]: Shouldn't there be a comma at the end of this line?

**Comment by untergeek** [comment]
Other than the missing comma, LGTM™

**Comment by purbon** [comment]
fixes :-P going to do the updates for systemd also here, 👍

**Comment by untergeek** [comment]
Cool.  Will wait for further response.

**Comment by purbon** [comment]
yeah, thanks! better to make all changes more compacted 👍, thanks for understanding.

**Comment by purbon** [comment]
@untergeek finally finished checking all distros, also I removed debian-7 as java8 is not going there.

**Comment by purbon** [comment]
let me know if there is any pending thoughts to be fix here.

**Comment by untergeek** [comment]
LGTM™

**Pull request closed without merging**